tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -423,6 +424,33 @@ public async Task ConfidentialClientUsingAdfsAsync()
             }
         }
 
+        [TestMethod]
+        public async Task ClientCreds_And_ADFS_LogRequestUri_OnError_Async()
```

Comment:
LGTM, but an AAD authority test is preferred

Reply:
Oh ok, I had a question on if I was going in the right direction but it seems so, I can update the test